src/plat/allwinnerA20/config.cmake

```diff
@@ -8,6 +8,13 @@ cmake_minimum_required(VERSION 3.7.2)
 
 declare_platform(allwinnerA20 KernelPlatformAllwinnerA20 PLAT_ALLWINNERA20 KernelSel4ArchAarch32)
 
+config_option(
+    KernelPlatformAllwinnerA20ArchTimer ARM_ALLWINNER_A20_ARCH_TIMER
```

Comment:
I wonder, is this an Alwinner A20 specific thing or is this something that could be generic for all ARM platforms since this is about the  "generic timer"?

Reply:
I think I probably gave a bad naming. This configuration just gives a choice for the kernel to use either the AllwinnerA20 specific timer (currently using) or the ARM generic timer. And we already have a generic timer implementation for all ARM platforms.